pkg/storegateway/bucket_stores.go

```diff
@@ -487,6 +487,7 @@ func (u *BucketStores) getOrCreateStore(userID string) (*store.BucketStore, erro
 		u.syncDirForUser(userID),
 		newChunksLimiterFactory(u.limits, userID),
 		newSeriesLimiterFactory(u.limits, userID),
+		store.NewBytesLimiterFactory(0),
```

Comment:
I assume this new parameter is due to new Thanos dependency. Does this parameter means "unlimited" and is the "old behaviour"? Mainly asking for confirmation :)

Reply:
Unlimited